lib/radius/cli/puma_dev.rb

```diff
@@ -0,0 +1,243 @@
+# frozen_string_literal: true
+
+require "dotenv"
+require_relative "helpers"
+
+module Radius
+  module Cli
+    # CLI command for setting up puma-dev for local HTTPS development
+    class PumaDev < Thor::Group
+      include Helpers
+
+      DESCRIPTION = "Configure current app for local HTTPS development through puma-dev"
+
+      desc DESCRIPTION
+
+      class_option :cert,
+                   type: :boolean,
+                   default: false,
+                   desc: "Add Puma-dev CA as trusted user cert to login keychain"
+
+      class_option :force,
+                   type: :boolean,
+                   default: false,
+                   desc: "Force overwriting links and ENV settings"
+
+      class_option :setup,
+                   type: :boolean,
+                   default: false,
+                   desc: "Initial puma-dev setup"
+
+      class_option :verbose,
+                   type: :boolean,
+                   default: false,
+                   desc: "Print progress to standard out"
+
+      def toggle_verbosity
+        $stdout.reopen(IO::NULL) unless options[:verbose]
+      end
+
+      def pow_conflict_check
+        return unless system("which pow 1>/dev/null 2>&1")
+
+        abort <<~INFO
+
+          CONFLICT: aborting puma-dev setup because pow is installed
+          Please uninstall pow then setup puma-dev.
+          See https://github.com/puma/puma-dev
+        INFO
+      end
+
+      def puma_dev_check
+        @cmd = `which puma-dev`.chomp
+        abort "Unable to configure puma-dev: not installed" unless $CHILD_STATUS.success?
```

Comment:
Why are we using `$CHILD_STATUS` as opposed to `$?`

Reply:
😰 b/c I always forget what `$?` is when I haven't worked with code that shells out code for a few weeks. The [`English`](https://ruby-doc.org/stdlib-2.5.3/libdoc/English/rdoc/English.html) stdlib defines named global alias for many of the more cryptic Perl-ish globals.